pkg/handler/iam_handler.go

```diff
@@ -106,11 +109,29 @@ func NewIAMHandler(ctx context.Context, organizationsClient, foldersClient, proj
 	return h, nil
 }
 
+// Cleanup removes expired IAM bindings added by AOD from the IAM policies of the resources in the request.
+func (h *IAMHandler) Cleanup(ctx context.Context, r *v1alpha1.IAMRequest) (nps []*v1alpha1.IAMResponse, retErr error) {
+	for _, p := range r.ResourcePolicies {
+		// time.Now() is a dummy parameter used to match the function signature.
+		np, err := h.handlePolicy(ctx, p, time.Now(), h.cleanupExpiredBindings)
```

Comment:
I don't think we should ignore cleanup error if it's an explicit cleanup call.

Reply:
Updated